src/promptflow-tools/promptflow/tools/common.py

```diff
@@ -227,9 +239,9 @@ def wrapper(*args, **kwargs):
                         )
                         print(msg, file=sys.stderr)
                     time.sleep(retry_after_seconds)
-                except OpenAIError as e:
-                    # For other non-retriable errors from OpenAIError,
-                    # For example, AuthenticationError, APIConnectionError, InvalidRequestError, InvalidAPIType
+                except APIError as e:
```

Comment:
I think here should continue to use `OpenAIError`, because it is the parent class of `APIError`.

https://github.com/openai/openai-python/blob/9f828cb1ea7c22a6388fcd71b5de1a0dde3a9a9f/src/openai/_exceptions.py#L24C7-L24C18

Reply:
replaced